### PR TITLE
OpenTelemetry collector 0.42.0

### PR DESCRIPTION
**Pull request by rkukura**
Signed-off-by: Robert Kukura <rkukura@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+This repository configures a build of the OpenTelemetry Collector with the supported components for a Red Hat OpenShift distributed tracing product release.
+
+## Updating and Testing
+
+1. Update Makefile and manifest.yaml to select the desired upstream version and component selection for the product release. 
+1. If the component selection has changed, update test-config.yaml to configure all supported extensions, receivers, processors, and exporters, and to include these in the service pipeline. 
+1. Run `make build`
+1. Run `./_build/otelcol --config test-config.yaml` and make sure that otelcol starts without any errors or warnings, then kill it.
+1. Create a merge request with the changes, including changes in the _build directory.
```
Comment (rubenvp8510) [inline README.md:11]: This should be a pull request, not merge request.

File: manifest.yaml [hunk 2]
```diff
@@ -2,33 +2,37 @@ dist:
   module: github.com/open-telemetry/opentelemetry-collector-releases/core
```
Comment (rubenvp8510) [inline manifest.yaml:2]: Could you change this to point to our repo?

File: README.md (hunk 1 above)
Comment (rkukura) [reply README.md]: Done

File: manifest.yaml (hunk 2 above)
Comment (rkukura) [reply manifest.yaml]: Done

**Review by rubenvp8510 — APPROVED** [review]

**Pull request merged**